config/locales/telephony/en.yml

```diff
@@ -10,25 +10,25 @@ en:
         %{app_name}: Your one-time code is %{code}. It expires in %{expiration} minutes. Don't share this code with anyone.
 
         @%{domain} #%{code}
-      voice: Hello! Your %{app_name} one-time code is, %{code}. Your one-time code is,
-        %{code}. Again, your one-time code is %{code}. This code expires in
-        %{expiration} minutes.
+      voice: Hello! Your %{app_name} 6-%{format_type} one-time code is, %{code}. Your
+        6-%{format_type} one-time code is, %{code}. Again, your 6-%{format_type}
+        one-time code is %{code}. This code expires in %{expiration} minutes.
```

Comment:
I'm trying to track down the content for this, do you know where it lives?

I found a Figma design in LG-8352, and it only adds the "6-digit" prefix before the first time it's announced.

Can we include the content in the implementation ticket?

Reply:
Ah yea. I think I found it from a Thread, I thought it was for the entire thing since the content repeats/ the first time its said can get cut off in a voicemail so we would run into the same issue with users still not knowing how long a code to expect.